lib/composer.js

```diff
@@ -14,11 +15,16 @@ import MagicParser from './parsers/magic-parser'
 export default class Composer extends Disposable {
   disposables = new CompositeDisposable()
   cachedBuildStates = new Map()
+  cachedDicy = new Map()
 
   constructor () {
     super(() => this.disposables.dispose())
     this.disposables.add(atom.config.onDidChange('latex', () => {
       this.rebuildCompleted = new Set()
+      const dicy = this.cachedDicy.values().next().value
+      if (dicy) {
+        dicy.updateOptions(this.getDicyOptions(), true)
+      }
```

Comment:
`updateOptions` writes the options by writing them into `.dicy.yaml` in the user's home directory. All  builder instances will notice the change in timestamp/hash and update their options in response. So it doesn't really matter which one it get's called on.

Conceptually, it should probably be a static member function, but I think I had to call some instance functions in it.